src/redux/actions/projectActions.ts

```diff
@@ -1,26 +1,38 @@
-import { Dispatch, Action } from "redux";
+import { Action, Dispatch } from "redux";
 import ProjectService from "../../services/projectService";
 import { ActionTypes } from "./actionTypes";
 import { AssetService } from "../../services/assetService";
 import { ExportProviderFactory } from "../../providers/export/exportProviderFactory";
 import {
-    IProject, IAsset, IAssetMetadata, IApplicationState,
-    ErrorCode, AppError } from "../../models/applicationState";
-import { createPayloadAction, IPayloadAction, createAction } from "./actionCreators";
-import { IExportResults } from "../../providers/export/exportProvider";
+    AppError,
+    ErrorCode,
+    IApplicationState,
+    IAsset,
+    IAssetMetadata,
+    IProject,
+} from "../../models/applicationState";
+import { createAction, createPayloadAction, IPayloadAction } from "./actionCreators";
+import { ExportAssetState, IExportResults } from "../../providers/export/exportProvider";
 import { appInfo } from "../../common/appInfo";
 
 /**
  * Actions to be performed in relation to projects
  */
 export default interface IProjectActions {
     loadProject(project: IProject): Promise<IProject>;
+
     saveProject(project: IProject): Promise<IProject>;
+
     deleteProject(project: IProject): Promise<void>;
+
     closeProject(): void;
+
     exportProject(project: IProject): Promise<void> | Promise<IExportResults>;
+
     loadAssets(project: IProject): Promise<IAsset[]>;
+
     loadAssetMetadata(project: IProject, asset: IAsset): Promise<IAssetMetadata>;
+
     saveAssetMetadata(project: IProject, assetMetadata: IAssetMetadata): Promise<IAssetMetadata>;
```

Comment:
nit: Any reason for adding the extra spaces? I think when exporting an interface it is more readable and consistent with our other formatting to leave them out.

Reply:
oh autoformat on webstorm did that, I can undo it